README.md

```diff
@@ -1,70 +1,62 @@
-# Getting Started with Create React App
+# Book store
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This project will lay foundations for the Bookstore website.
+React and Redux app was built and set up routing using React Router.
+the project built-in :
 
-## Available Scripts
+React
+Redux
+webpack
+babelrc
+linter
 
-In the project directory, you can run:
+## Live Demo
 
-### `npm start`
+[link to Bookstore](https://relaxed-bartik-b47c93.netlify.app/)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+To get a local copy up and running follow these simple example steps.
 
-### `npm test`
+### Prerequisites
+npm/yarn installed
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `npm run build`
-
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `npm run eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
-
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
-
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+### Setup
+clone the repo: git clone git@github.com:Marcraphael12/Marc-the-magician.git
+jump to the active branch: git checkout setup-react
+### Install
+in the terminal, run: npm installor npm i
+### Usage
+now run npm startto start the app
+### Run tests
+Author only
+### Deployment
+This project was deployed on Netlify 
```

Comment:
- [OPTIONAL] In this section I think you can use `- before the line`, `**text**`, or `_text_` to separate commands and their descriptions from each other. Because it makes more clear and easier to read and understand for users of your project.